package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@gamechanger/eslint-plugin",
-    "version": "0.0.9",
+    "version": "0.0.10",
```

Comment:
should we make it a major version release 1.0.0 since we are upgrading to major release of TS

Reply:
It depends. I'm not sure what the impact the linter has on the project that uses it. If the project won't have any issues of compatibility, this shouldn't affect it and it doesn't need to be a major version bump. Although, the versioning here doesn't seem to follow semver anyways. (this is a patch update and not an actual minor update)

Reply:
Yeah this repo hasn't really followed semver, but I think this is also a 'supports' and not 'requires' so a patch bump makes sense.